plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLPreferenceConstants.java

```diff
@@ -99,11 +96,13 @@ public static EmptyScriptCloseBehavior getByName(String name) {
     public static final String SQLEDITOR_CLOSE_COMMENTS                = "SQLEditor.closeComments";
     public static final String SQLEDITOR_CLOSE_BLOCKS                  = "SQLEditor.closeBlocks";
 
+    public static final String SMART_WORD_ITERATOR                     = "SQLEditor.smartWrdIterator";
```

Comment:
Is this a typo (`Wrd` instead of `Word`) or is there some kind of (length?) restriction?

Reply:
Yep :)